### PR TITLE
[UPGRADE] MAVEN dependencies

### PR DESCRIPTION
**Pull request by vttranlina**

### CODE REVIEW

**Review by quantranhong1999 — COMMENTED** [review]
Read it so far.

File: server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/WebSocketContract.scala [hunk 1]
```diff
@@ -191,6 +191,7 @@ trait WebSocketContract {
         .body
 
     assertThatJson(response.toOption.get)
+      .whenIgnoringPaths("detail")
```
Comment (chibenwa) [inline server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/WebSocketContract.scala:194]: What is the point of ignoring detail to later assert it? I think this change breaks the test.

Comment (vttranlina) [reply server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/WebSocketContract.scala:194]: No, it doesn't break the test. The details are overly verbose, generated by the Play JSON library.

Before
`"The request was successfully parsed as JSON but did not match the type signature of the Request object: List((,List(JsonValidationError(List(Unrecognized token 'The': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')
 at [Source: (String)"The quick brown fox"; line: 1, column: 4]),List()))))"`

After

`"The request was successfully parsed as JSON but did not match the type signature of the Request object: List((,List(JsonValidationError(List(Unrecognized token 'The': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 4]),List()))))" `

Comment (vttranlina) [reply server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/WebSocketContract.scala:194]: let me add one more assert for more clear

```
    assertThatJson(response.toOption.get)
      .node("detail")
      .isString.contains("The request was successfully parsed as JSON but did not match the type signature of the Request object")
```

**Review by Arsnael — APPROVED** [review]

**Review by quantranhong1999 — APPROVED** [review]

**Pull request merged**